unit_test/sparse/Test_Sparse_gauss_seidel.hpp

```diff
@@ -248,14 +248,120 @@ void test_gauss_seidel(lno_t numRows, size_type nnz, lno_t bandwidth, lno_t row_
   //device::execution_space::finalize();
 }
 
+template <typename scalar_t, typename lno_t, typename size_type, typename device>
```

Comment:
How do you test performance ? Is there a perf-test ?

Reply:
Yes, there is a performance test KokkosSparse_gs.cpp that runs on a given matrix with point-coloring SGS, and then cluster SGS with varying cluster sizes.